comid/measurement.go

```diff
@@ -436,7 +437,10 @@ func (o Mval) Valid() error {
 		o.UEID == nil &&
 		o.UUID == nil &&
 		o.IntegrityRegisters == nil {
-		return fmt.Errorf("no measurement value set")
+
+		if o.Extensions.IsEmpty() {
```

Comment:
Why not include this in the conjunction? I don’t see a reason for the nested if.

Reply:
I uses nested if statements for simpler view of the conditions. However, if using a conjunction is preferred for better readability, I can change it accordingly.

Reply:
I would prefer it. I don’t think there’s anything about this condition that needs special attention

Reply:
Thanks for the feedback!  it's cleaner with conjunctions. I've updated the code